sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -538,6 +493,12 @@ export type TimeFieldValue = {
     value?: string;
 } & CommonFieldValue;
 
+// @public
+export class TrainCustomFormModelError extends Error {
```

Comment:
curious why we have the distinction of errors between RecognizedForms and training?

Reply:
Maybe because I did them in two separate branches! I will update to use just one.

Reply:
sigh, the good name is already taken by the inner error.  I actually want `FormRecognizerError` to be the error type, and something else for the nested error information. `Error` is special in JS (counter part of Exception in .NET)

Reply:
I changed to throw a generic `Error` and use `FormRecognizerError` for error info returned by the service (similar to `TextAnalyticsError`).